src/main/webapp/app/exercises/programming/manage/instructions-editor/analysis/programming-exercise-instruction-analysis.service.ts

```diff
@@ -152,7 +152,7 @@ export class ProgrammingExerciseInstructionAnalysisService {
                     uniq,
                     filter((m) => !!m),
                     flatten,
-                )(match.split(',').map((m: string) => m.trim()));
+                )(match.split(/,(?![^(]*?\))/).map((m: string) => m.trim()));
```

Comment:
Can we get the same comment we have in the other places here, too please?

Reply:
Maybe we could (instead) define all those equal regexes directly as constant in some central place? But comment here would help if this is not possible or too difficult.